README.md

```diff
@@ -80,7 +81,7 @@ This software has been compiled and tested under PHP 7.0.20 and 7.1.6 using the
 3. Restart the Web server.
 
 ## Install (UNIX)
-The following instructions assume a clean environment and show how to install PHP 7.x, Microsoft ODBC driver, apache, and Microsoft PHP drivers on Ubuntu 15, 16, RedHat 7, Debian 8, and Mac OS X. 
+The following instructions assume a clean environment and show how to install PHP 7.x, Microsoft ODBC driver, apache, and Microsoft PHP drivers on Ubuntu 15, 16, RedHat 7, Debian 8, SUSE 12, and Mac OS X. 
```

Comment:
No more Ubuntu 15, and it's no longer called Mac OS X or OS X it's now xalled macOS (or Mac OS). Also, nit-picking but Apache should be capitalized